src/test/java/org/opensearch/security/http/OnBehalfOfAuthenticatorTest.java

```diff
@@ -267,7 +267,7 @@ public void testBearer() throws Exception {
         Map<String, String> expectedAttributes = new HashMap<>();
         expectedAttributes.put("attr.jwt.iss", "cluster_0");
         expectedAttributes.put("attr.jwt.sub", "Leonard McCoy");
-        expectedAttributes.put("attr.jwt.aud", "[ext_0]");
+        expectedAttributes.put("attr.jwt.aud", "[\"ext_0\"]");
```

Comment:
just for my knowledge, why we need tis change here? Is this to ensure the JSON representation of the collection, like a list for this case?

Reply:
Yes exactly. The existing parsing logic works, but some users refer to these attributes when constructing DLS clauses. For example:

```
"dls": """{
  "terms": {
    "group_permissions.keyword": ${attr.jwt.opensearch_roles}
  }
```

With the existing parsing of these attributes, this DLS clause above would produce malformed JSON. By ensuring that the attribute parsing produces a json-parseable list it would avoid the error. See the description of the attached issue which has a full example: https://github.com/opensearch-project/security/issues/4267